Themes/Inspired by Things (Dark).theme.json

```diff
@@ -0,0 +1,119 @@
+{
+    "name": "Inspired by Things",
+    "style": "Dark",
+    "author": [
+        {
+            "name": "Matt Stratford",
+            "twitter": "@mattstratford"
+        }
+    ],
+    "version": "0.2.1",
+
+    "editor": {
+        "backgroundColor": "#312E37",
+        "caretColor": "#FE3A6F",
+        "selection": {
+            "backgroundColor": "#6BAEF8",
+            "unfocusedBackgroundColor": "#CAE2FE"
+        },
+        "highlight": {
+            "backgroundColor": "#5694F9",
+            "unfocusedBackgroundColor": "#F04E98"
+        }
+    },
+
+    "savedSearchesSidebar": {
+        "color": "#FFFFFF",
+        "backgroundColor": "#2C2933",
+        "highlightedBackgroundColor": "#FED530",
+        "border": false,
+        "iconColors": {
+            "default": "#6BAEF8"
+        }
+    },
+
+    "resultsList": {
+        "color": "#D4D2D5",
+        "backgroundColor": "#2C2933",
+        "alternateBackgroundColor": "#222326",
+        "selection": {
+            "color": "#2C2933",
+            "backgroundColor": "#F3D331",
+            "unfocusedColor": "#DDDCDE",
+            "unfocusedBackgroundColor": "#4B4850"
+        },
+        "border": "#6BAEF8",
+    },
+
+    "styles": {
+        "base": {
+            "color": "#FFFFFF",
+        },
+
+        "h1": {
+            "font": { "style": "bold" }
+        },
+        "h2": {
+            "font": { "style": "bold" }
+        },
+        "h3": {
+            "font": { "style": "bold" }
+        },
+        "h4": {
+            "font": { "style": "bold" }
+        },
+        "h5": {
+            "font": { "style": "bold" }
+        },
+        "h6": {
+            "font": { "style": "bold" }
+        },
+
+        "horizontalRule": {
+            "color": "#ABAFB4"
+        },
+
+        "codeBlock": {
+            "backgroundColor": "#3AA59A",
+            "font-family":"iA Writer Mono S, Courier, sans-serif",
```

Comment:
Font families don't have fallbacks as they have with CSS, so you can just pick the iA Writer ones and omit the rest (also for the font families of the light variant)